modelexpress_common/proto/refit.proto

```
@@ -114,7 +130,10 @@ message WeightVersionShard {
   uint64 total_bytes = 5;
   string manifest_digest = 6;
   string manifest_endpoint = 7;
-  string transport = 8;
+}
```

Buf (1.72.0)

[error] Previously present field "8" with name "transport" on message "WeightVersionShard" was deleted without reserving the name "transport".

(FIELD_NO_DELETE_UNLESS_NAME_RESERVED)